scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
```

Comment:
:stop_sign: **Security Vulnerability**: Hardcoded sensitive information detected. Storing API keys, passwords, and database credentials directly in the source code is a severe security risk[^1]. These should be moved to a secure configuration management system or environment variables.
```suggestion
// Remove hardcoded credentials
// Use environment variables or a secure configuration management system
const apiKey = process.env.API_KEY
const password = process.env.PASSWORD
const dbConnection = process.env.DB_CONNECTION
```

[^1]: CWE-798: Use of Hard-coded Credentials - https://cwe.mitre.org/data/definitions/798.html